model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/JpaMapStorageProviderFactory.java

```diff
@@ -288,8 +288,10 @@ protected EntityManager getEntityManager() {
             if (lockTimeout != null) {
                 em.unwrap(SessionImpl.class)
                         .doWork(connection -> {
-                            PreparedStatement preparedStatement = connection.prepareStatement("SET LOCAL lock_timeout = '" + lockTimeout + "';");
-                            preparedStatement.execute();
+                            try (PreparedStatement preparedStatement = connection.prepareStatement("SET LOCAL lock_timeout = ?")) {
+                                preparedStatement.setString(1, String.valueOf(lockTimeout));
```

Comment:
I tried to use parameters here, however, it seemed that something (Postgres/Hibernate) support paremeters only for some SQL commands and the way I understood it `SET` is not between them. But would be great if you were able to make it work.

Reply:
> I tried to use parameters here, however, it seemed that something (Postgres/Hibernate) support parameters only for some SQL commands and the way I understood it SET is not between them. But would be great if you were able to make it work.

I've seen it work for CRDB, and now see it failing for PostgreSQL on GHA. Can only try it locally next week again. Will add comment that parameters won't work here. 

BTW: This is plain PostgreSQL JDBC, no Hibernate involved here as we're asking for a direct JDBC connection.

Reply:
Ahh, sure, that makes sense. Thanks for the explanation. I should have added the comment myself, sorry. Good luck with the conference btw. Hope you have a pleasant journey!

Reply:
Found a way with a prepared statement and a parameter. Build is now green with PostgreSQL, will test on Monday with CRDB, and then ask you for a review.